Spigot-API-Patches/0218-Return-chat-component-with-empty-text-instead-of-thr.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: CDFN <codefun@protonmail.com>
+Date: Tue, 7 Jul 2020 17:53:23 +0200
+Subject: [PATCH] Return chat component with empty text instead of throwing
+ exception
+
+
+diff --git a/src/main/java/org/bukkit/inventory/InventoryView.java b/src/main/java/org/bukkit/inventory/InventoryView.java
+index 14346d83bc99581b18e53d19af03708c0bf22cf7..98d8242122dff5811e689143cb26aaadd4b1d064 100644
+--- a/src/main/java/org/bukkit/inventory/InventoryView.java
++++ b/src/main/java/org/bukkit/inventory/InventoryView.java
+@@ -449,7 +449,7 @@ public abstract class InventoryView {
+     /**
+      * Get the title of this inventory window.
+      *
+-     * @return The title.
++     * @return The title or empty string when title is null.
```

Comment:
This should get a `// Paper` tag on it. Just at the end of the line.

Reply:
The javadoc is HTML, so:
```suggestion
+     * @return The title or empty string when title is {@code null}. <!-- Paper -->
```